drools-wb-services/drools-wb-verifier/drools-wb-verifier-core/src/main/java/org/drools/workbench/services/verifier/core/checks/DetectConflictingRowsCheck.java

```diff
@@ -47,15 +58,16 @@ public void check() {
     }
 
     @Override
-    public boolean isActive( final CheckWhiteList whiteList ) {
-        return whiteList.getAllowedCheckTypes()
-                .contains( CHECK_TYPE );
+    public boolean isActive( final CheckConfiguration whiteList ) {
```

Comment:
`WhiteList` class was moved to `CheckConfiguration` class, is it intended to keep attribute names (`whiteList`) according to old class name?